services/ad-tech/src/lib/interest-group-helper.ts

```diff
@@ -35,16 +35,18 @@ export interface InterestGroupAd {
   // REQUIRED FIELDS
   /** Main creative URL. */
   renderURL: string;
+  // OPTIONAL FIELDS
   /** Custom ad metadata stored by ad-tech. */
-  metadata: {
+  metadata?: {
     /** Hostname of the advertiser. */
     advertiser: string;
```

Comment:
can we leave a comment that advertiser and adType are required for the demo only, and not for the actual API usage?  i don't want people being confused that they need to define these fields.